src/main/webapp/i18n/de/result.json

```
@@ -20,20 +20,24 @@
             "detail": {
                 "title": "Bewertung"
             },
-            "noResult": "Kein gewertetes Ergebnis",
+            "noResult": "Kein Ergebnis",
+            "noGradedResult": "Kein gewertetes Ergebnis",
             "resultString": "Ergebnistext",
             "resultStringBuildFailed": "Build gescheitert",
             "resultStringBuildSuccessfulNoTests": "Build erfolgreich, keine Tests ausgeführt",
             "resultStringBuildSuccessfulTests": "{{ numberOfTestsPassed }} von {{ numberOfTestsTotal }} bestanden",
             "resultStringNonProgramming": "{{ relativeScore }}%, {{ points }} Punkte",
             "resultStringProgramming": "{{ relativeScore }}%, {{ buildAndTestMessage }}, {{ points }} Punkte",
             "resultStringProgrammingCodeIssues": "{{ relativeScore }}%, {{ buildAndTestMessage }}, {{ numberOfIssues }} Issues, {{ points }} Punkte",
+            "resultStringProgrammingShort": "{{ relativeScore }}%, {{ buildAndTestMessage }}",
+            "resultStringShort": "{{ relativeScore }}%",
             "completionDate": "Abgabedatum",
             "successful": "Erfolgreich",
             "buildArtifact": "Build-Artefakt",
             "buildError": "Build Fehler:",
-            "score": "Bewertung",
+            "score": "Bewertung {{score}}%",
             "scorePercentage": "Bewertung (in %)",
+            "scoreWithPoints": "Score  {{score}}, {{points}} points",
```

```suggestion
            "scoreWithPoints": "Bewertung {{score}}%, {{points}} Punkte",
```
- English text in the German translation file
- With percent sign or without? The `score` translation two lines up has the `%`.